<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>options</class>
 <widget class="QDialog" name="options">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>497</width>
    <height>325</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Options...</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="QTabWidget" name="generalTab">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="generalTabLayout">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0" colspan="2">
          <widget class="QGroupBox" name="processingMethodBox">
           <property name="title">
            <string>Methode de traitement :</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="4" column="0">
             <widget class="QRadioButton" name="afterPassage">
              <property name="text">
               <string>Apres passage</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="7" column="0">
             <widget class="QRadioButton" name="fixedDate">
              <property name="text">
               <string>Date fixe</string>
              </property>
             </widget>
            </item>
            <item row="11" column="0">
             <widget class="QSpinBox" name="fixedDateInt">
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>31</number>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QSpinBox" name="toleranceInt">
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="accelerated">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="10" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Choisir la date du traitement mensuel :</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Indiquer le nombre de jours tolerés entre le dernier traitement et le passage sur la sation :</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="dayNumberBox">
         <property name="title">
          <string>Dépassement de délais après le dernier traitement (stations hors bulletin) :</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="timeOutDayLabel">
            <property name="text">
             <string>Indiquer le nombre de jour maximim entre deux traitements :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="timeOutDayInt">
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="graphiqueTab">
      <attribute name="title">
       <string>Graphique</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QGridLayout" name="graphiqueTabGridLayout">
         <item row="4" column="1">
          <widget class="QPushButton" name="bulletinColor">
           <property name="text">
            <string>Modifer</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="processingColor">
           <property name="text">
            <string>Modifier</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QPushButton" name="defautProcessing">
           <property name="text">
            <string>Defaut</string>
           </property>
          </widget>
         </item>
         <item row="4" column="3">
          <widget class="QPushButton" name="defautBuletin">
           <property name="text">
            <string>Defaut</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1" colspan="3">
          <widget class="QLabel" name="bulletinColorLabel">
           <property name="sizeIncrement">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Couleur des stations du bulletin :</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="3">
          <widget class="QLabel" name="stationColorLabel">
           <property name="sizeIncrement">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Couleur des stations à traiter :</string>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QFrame" name="buletinColorView">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QFrame" name="processingColorView">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QPushButton" name="timeOutDayColor">
           <property name="text">
            <string>Modifier</string>
           </property>
          </widget>
         </item>
         <item row="6" column="3">
          <widget class="QPushButton" name="defautTimeOutDay">
           <property name="text">
            <string>Defaut</string>
           </property>
          </widget>
         </item>
         <item row="7" column="2">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="6" column="2">
          <widget class="QFrame" name="timeOutDayColorView">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
          </widget>
         </item>
         <item row="5" column="1" colspan="3">
          <widget class="QLabel" name="timeOutColorLabel">
           <property name="text">
            <string>Couleur des stations avec depassement du délais de traitement :</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="adminTab">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <attribute name="title">
       <string>Gestionnaire</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QGroupBox" name="AuthentificationBox">
         <property name="title">
          <string>Activer le mode gestionnaire</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="2" column="1" colspan="2">
           <widget class="QCheckBox" name="showMdpCheckBox">
            <property name="text">
             <string>Afficher le texte</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="3">
           <widget class="QLineEdit" name="mdpLigneEdit">
            <property name="maxLength">
             <number>255</number>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="4" column="2">
           <widget class="QPushButton" name="delAdminButton">
            <property name="text">
             <string>Suprimer</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QPushButton" name="checkAdminButton">
            <property name="text">
             <string>Verifier</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="CheckLabel">
            <property name="font">
             <font>
              <pointsize>11</pointsize>
              <weight>75</weight>
              <italic>true</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>options</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>options</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
